node_modules/oae-admin/tenants/css/tenants.css

```diff
@@ -13,32 +13,89 @@
  * permissions and limitations under the License.
  */
 
-#tenants-container .table-striped td.tenants-visit {
-    width: 20px;
+#tenants-search {
+    margin-bottom: 5px;
+    text-align: right;
 }
 
-#tenants-container .table-striped td.tenants-visit i {
+#tenants-search > form {
+    display: inline-block;
+    margin-bottom: 5px;
+    min-width: 300px;
+}
+
+#tenants-table li {
+    background-color: #FFF;
+    border-bottom: solid 1px #DDD;
+}
+
+#tenants-table li > div {
+    line-height: 35px;
+    padding: 8px;
+}
+
+#tenants-table-header li {
+    font-weight: bold;
+}
+
+#tenants-table-header li,
+#tenants-table-rows li:nth-child(2n) {
+    background-color: #F8F8F8;
+}
+
+#tenants-table-rows .alert {
+    margin-top: 10px;
+}
+
+#tenants-table .tenants-col {
```

Comment:
I don't really understand how we're managing to get the headers to align with the actual columns without using `display: table`/`display: table-row`/`display: table-cell`